src/plots/ternary/layout/attributes.js

```diff
@@ -0,0 +1,24 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
```

Comment:
I see that you put `attributes` and `defaults` modules in `plots/ternary/layout` similar to how `geo` is setup whereas `index.js` and `ternary.js` are in `plots/ternary/`.

Do you like that partition between *root* and *layout* modules?

If I had do it all over again, I would simply put every subplot file in `plots/<subplot>/`. No need for that `layout/` subdirectory. What do you think?